pkg/distribution/configuration/parser.go

```diff
@@ -92,6 +92,9 @@ func NewParser(prefix string, envs []string, parseInfos []configuration.Versione
 
 	for _, env := range envs {
 		envParts := strings.SplitN(env, "=", 2)
+		if len(envParts) < 2 {
```

Comment:
here would be better to check that is exactly 2 the length? Should it error out, or just avoiding appending is enough? do not know the use case

Reply:
SplitN should return exactly 2, but just in case somehow that isn't the case I used `<`.

Usecase here is to split and parse value like `"REGISTRY_STORAGE_S3_ACCESSKEYID=AKIAIOSFODNN7EXAMPLE"`
example at https://github.com/migtools/udistribution/blob/main/pkg/client/client_test.go#L45

Happy to switch to `==`

Reply:
looked into `SplitN` docs https://pkg.go.dev/strings#SplitN, max value will be 2, but can be less, so I am ok with the current check